test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -449,6 +450,40 @@ def read_conf_file
   end
 end
 
+shared_examples_for 'an Agent with python3 enabled' do
+  it 'restarts after python_version is set to 3' do
+    conf_path = ""
+    if os != :windows
+      conf_path = "/etc/datadog-agent/datadog.yaml"
+    else
+      conf_path = "#{ENV['ProgramData']}\\Datadog\\datadog.yaml"
+    end
+    f = File.read(conf_path)
+    confYaml = YAML.load(f)
+    confYaml["python_version"] = 3
+    File.write(conf_path, confYaml.to_yaml)
+
+    output = restart
```

Comment:
Though this is enough to check the agent comes up, I'm wondering if we shouldn't check the `info` page (status), to actually verify the correct interpreter is being used. Feel free to disregard if you think that's beyond the scope of what we're testing.